src/script/lua_api/l_noise.h

```diff
@@ -160,4 +160,40 @@ class LuaPcgRandom : public ModApiBase {
 	static void Register(lua_State *L);
 };
 
+
+/*
+	LuaSecureRandom
+*/
+class LuaSecureRandom : public ModApiBase {
+private:
+	PcgRandom m_rnd;
+	#define SEC_RAND_BUFF_MAX 128
```

Comment:
Does this really need to be a constant? Why not just use the number literal in the declaration of m_rand_buff, and then use sizeof(m_rand_buff) anywhere this macro would be used?

Besides, it's not supposed to be indented.